### PR TITLE
Dynamic servers address selected

### PR DESCRIPTION
**Pull request by mrpalide**
Did you run `make format && make check`?

Fixes: Issue [426](https://github.com/SkycoinPro/skywire-services/issues/426) in SkycoinPro/skywire-services

 Changes:
- [x] Create new servers.yml file that should fetch from skycoin.com or load from local file
- [x] Add two variable to skywire-config.json
    - `ServersListAddress` : address for fetching servers list data
    - `IsTest` : checking config is for test or production
- [x] Add new flag for skywire-visor `--local-server` that force to connect local servers
- [x] Check timezone automate for choosing best server 

How to test this PR:
Every command like before, just for choosing local servers manually, add `--local-server true` to skywire-visor running command like this:

> `sudo ./skywire-visor -c skywire-config.json --local-server true`

### CODE REVIEW

File: servers.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+test:
```
Comment (jdknives) [inline servers.yml:1]: Does yml allow comments? If it does, we should add some basic documentation for users how to deal with this file and explain the non-obvious fields such as capacity.

File: pkg/skyenv/values.go [hunk 2]
```diff
@@ -8,8 +8,9 @@ import (
 
 // Constants for skywire root directories.
 const (
-	DefaultSkywirePath = "."
-	PackageSkywirePath = "/opt/skywire"
+	DefaultSkywirePath        = "."
+	PackageSkywirePath        = "/opt/skywire"
+	DefaultServersListAddress = "http://servers-list.skycoin.com"
```
Comment (jdknives) [inline pkg/skyenv/values.go:14]: I will speak to Murphy about the actual name of the domain. Add a TODO here for now please.

Comment (mrpalide) [reply pkg/skyenv/values.go:14]: Added

File: servers.yml (hunk 1 above)
Comment (mrpalide) [reply servers.yml:1]: It's possible for comments, but I don't know add what and how docmuentation.

**Comment by alexadhy** [comment]
Can we use toml / json instead? saw that the `servers.yml` config doesn't contain that many children / indentations. Just my 2 cents, correct me if I'm wrong.

**Comment by mrpalide** [comment]
Because of some problems, this PR continues on #918.

**Pull request closed without merging**